### PR TITLE
PIN-3493 Purpose Process Migration

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

File: src/test/scala/it/pagopa/interop/purposeprocess/SpecData.scala [hunk 1]
```diff
@@ -135,16 +149,18 @@ object SpecData {
...
-      if (tenantKind == TenantKind.PA) validOnlySchemaRiskAnalysis2_0 else validOnlySchemaRiskAnalysis1_0
+      if (tenantKind == PersistentTenantKind.PA) validOnlySchemaRiskAnalysis2_0 else validOnlySchemaRiskAnalysis1_0
     RiskAnalysisValidation
       .validate(validOnlySchemaRiskAnalysis, true)(tenantKind)
       .toOption
       .get
   }
 
-  def validOnlySchemaManagementRiskAnalysis(tenantKind: TenantKind): PurposeManagement.RiskAnalysisForm = {
+  def validOnlySchemaManagementRiskAnalysis(tenantKind: PersistentTenantKind): PurposeManagement.RiskAnalysisForm = {
```
Comment (galales) [inline src/test/scala/it/pagopa/interop/purposeprocess/SpecData.scala]: is this still used?

**Review by galales — APPROVED** [review]
thanks for the refactor

**Comment by TonioGela** [comment]
Do not merge it until tomorrow please

**Review by TonioGela — APPROVED** [review]
This was massive!

**Review by beetlecrunch — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**